src/main/augurNodeController.js

```diff
@@ -67,36 +67,52 @@ AugurNodeController.prototype.setWindow = function (window) {
 }
 
 AugurNodeController.prototype.initializeDatabaseAndStartServer = function () {
-    // Eventually it would be nice to keep these around
     try {
-        if (fs.existsSync(this.augurDbPath)) {
-            fs.unlinkSync(this.augurDbPath);
-        }
-        this.db = Knex({
-            client: "sqlite3",
-            connection: {
-                filename: this.augurDbPath,
-            },
-            acquireConnectionTimeout: 5 * 60 * 1000,
-            useNullAsDefault: true,
-            postProcessResponse: postProcessDatabaseResults,
-        });
-        this.db.migrate.latest({directory: path.join(__dirname, '../../node_modules/augur-node/build/migrations')}).then(function() {
-            try {
-                this.startServer();
-                this.startSyncProcess();
-                this.running = true;
-            } catch (err) {
-                console.error(err);
-                this.window.webContents.send("error", { error: err.toString() });
-            }
+        this.augur.connect({ ethereumNode: { http: this.networkConfig.http, ws: this.networkConfig.ws }, startBlockStreamOnConnect: false }, function () {
+            const networkId = this.augur.rpc.getNetworkID();
+            this.augurDbPath = path.join(this.appDataPath, `${networkId}-augur.db`)
```

Comment:
I wonder if putting the networkId after would be better from grouping? augur-${networkId}.db ? minor